### DISCUSSION TITLE
Bardak, Su, buz sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#206] [Zeka Sorusu]
Bir bardakta bulunan bir miktar suyun içine buz atıyorsunuz ve suyun seviyesini bardağa kalemle işaretliyorsunuz. Buz eridikten sonra su seviyesinde ne gibi bir değişme olur?

**Comment by HabilCcabbar** [comment]
Artar. Çünkü buz donmuş sudur ve erimiş buzda sudur.

**Comment by HabilCcabbar** [comment]
Aynı kalma ihtimali de var. Çünkü buz erirken su da buharlaşır.

**Comment by Furkanyildirm** [comment]
değişme olmaz çünkü su hacmine buz eklendikten sonra ölçülüyor buz eriyince değişiklik olmaz

**Comment by begumasyaburus** [comment]
(buzun yoğunluğu 0.9 , suyun yoğunluğu 1) (1 yoğunluk birimi=1 kap birimi)

Bir kap düşünelim içerisindeki su miktarı 2 birim olsun, buzu koyunca 2,9 birim artar suyun buharlaşmadığını da ele alırsak buz eriyince, suyun yüksekliği buzun kütlesi kadar artacaktır yani 3 birim olacaktır
